Makefile

```diff
@@ -254,6 +258,11 @@ start-stress-test: zetanode
 	@echo "--> Starting stress test"
 	cd contrib/localnet/ && $(DOCKER) compose --profile stress -f docker-compose.yml up -d
 
+start-solana-test: zetanode solana
```

Comment:
For simplicity we can have Solana test running as part of `start-e2e-test` in a separate runner like Bitcoin test

As Solana is only integrated for test for now, it would be acceptable to keep it like this but would be great to create an issue to track this change

Reply:
No, the solana local validator is quite CPU intensive and should not run by default.